bears/vcs/git/GitCommitBear.py

```diff
@@ -102,6 +105,70 @@ def get_host_from_remotes():
             netloc = urlparse(url)[1]
         return netloc.split('.')[0]
 
+    @staticmethod
+    def get_owner_from_url(url):
```

Comment:
We are looking for a library to parse git urls already,
IMO it would be much better to replace these functions by a library that does this already
For e.g have a look at [this](https://github.com/retr0h/git-url-parse) and [this](https://github.com/nephila/giturlparse)

Reply:
Most of these tasks are handled already by [IGitt](https://gitlab.com/gitmate/open-source/IGitt), I recommend using that.

Reply:
Yes. Could be done not an issue.

Reply:
All of this remote stuff, and issue fetching, is completely off base.

Anything to do with remotes must be removed.  Even guessing what `#1` is not necessarily in scope of this issue.  We have setting `body_close_issue_full_url` which could be required to be enabled for your feature to work. 

As you have now learnt, you can not use the GitLab API for issues.  How are you going to check the URL? :P